### ISSUE TITLE
not starting

### ISSUE THREAD

**Issue by Scorpion2606** [JamesWilko/Payday-2-BLT#150]
i installed fresh copy of the game and put BLT_release_r7_r18 in it.when i start the game for a second it shows in game in steam then close but game doesnt start. there is no mods added to it. Running on Win 7

**Comment by ghost** [comment]
Did you make sure to replace the IPHLPAPI.dll file in your PAYDAY 2 folder?

**Comment by Scorpion2606** [comment]
Yes, i copied mods folder and IPHLPAPI.dll file together from zip file. There isn't any previous to replace.

**Comment by ghost** [comment]
I've heard that the Windows 7 version is broken anyway, so you might just have to wait for a patch. IDK.
